lib/class-experimental-wp-widget-blocks-manager.php

```diff
@@ -330,6 +330,12 @@ public static function convert_blocks_to_widget( $blocks ) {
 				'blocks' => $blocks,
 			)
 		);
+		wp_register_widget_control(
```

Comment:
According to the commit, it seems like this was added to avoid missing widget control errors. The control is the edition interface of a widget, and it looks like widget controls are not mandatory, e.g., I can have a print hello widget, without any UI to edit it.
It would be interesting to check if the warnings also happen on other widgets without a control associated as it may be an existing core bug or it may be another problem in our Blocks Area Widget.
For now, I guess if this solution fixes the problem it is something we can use.

Reply:
Yeah I found it strange too.